src/transform-applier.js

```diff
@@ -522,56 +550,42 @@ const transformStrokeWidths = function (svgTag, windowRef, bboxForTesting) {
                     element.childNodes[i],
                     Matrix.compose(matrix, _parseTransform(element)),
                     strokeWidth,
-                    fill
+                    fill,
+                    stroke
                 );
             }
             element.removeAttribute('transform');
             element.removeAttribute('stroke-width');
             element.removeAttribute('fill');
+            element.removeAttribute('stroke');
         } else if (_isPathWithTransformAndStroke(element, strokeWidth)) {
```

Comment:
We should double check if this code is working for stuff besides paths with strokes, like shapes